scripts/run-all.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euo pipefail
+
+rm -f versions.env
+
+chmod +x scripts/get_versions.sh
+chmod +x scripts/version_check.sh
```

Comment:
Wondering why this is necessary, as you don't have to do set the other `.sh` as executable. I think it's possible to commit scripts as executable with git. Check [this](https://stackoverflow.com/a/40979016) if it doesn't work with Intellij natively